src/rt-app_parse_config.c

```diff
@@ -363,6 +365,44 @@ static char* create_unique_name(char *tmp, int size, const char* ref, long tag)
 	return tmp;
 }
 
+void initialise_external_library(rtapp_resource_t *library_struct)
```

Comment:
simply rdata maybe (as it contains more data that the one strictly associated with the library)?

Reply:
I am having difficulties understanding what the comment refers to. Can you please rephrase?

Reply:
Oh, sorry, simply a huge nitpick.. I'd s/library_struct/rdata/ :-)

Reply:
changed library_struct to rdata